enrolments/tests/serializers/test_class_create.py

```diff
@@ -0,0 +1,27 @@
+from django.core.exceptions import ValidationError
+from django.test import TestCase
+from unittest.mock import patch
+
+from accounts.models import User
+from enrolments.models import Class
+from enrolments.serializers import ClassCreateSerializer
+
+
+class ClassCreateSerializerTestCase(TestCase):
+    def setUp(self):
+        self.name = "Class 1"
+        self.days = [Class.MONDAY, Class.THURSDAY]
+        self.location = "Waterloo"
+        self.facilitator = User.objects.create(email="user@staff.com")
+
+        self.class_payload = {
+            "name": self.name,
+            "days": self.days,
+            "location": self.location,
+            "facilitator": self.facilitator.id,
+        }
+
+    @patch("enrolments.serializers.ClassCreateSerializer.create")
+    def test_class_serializer_create(self, mock_create):
+        serializer = ClassCreateSerializer(data=self.class_payload)
+        self.assertTrue(serializer.is_valid())
```

Comment:
could we also validate that a class is created with M&G in attendance? you can check out the test for the family post endpoint [here](https://github.com/uwblueprint/project-read-backend/blob/fbeeea154c5a99b7d24b71034c6fae161ac20602/registration/tests/serializers/test_family_detail_create.py#L66)!

Reply:
could we also add a small check that a Class object was created 🥺 sorry i didn't catch that earlier